app/templating/StringHelper.scala

```diff
@@ -47,8 +47,23 @@ trait StringHelper { self: NumberHelper =>
     }
   }
 
+  // Matches a lichess username with a '@' prefix (I hope.) Example: @ornicar
+  private val atUsernameRegex = "(?<=^|(?<=[^a-zA-Z0-9-_\\.]))@([A-Za-z]+[A-Za-z0-9]+)".r
```

Comment:
It also seems `[A-Za-z]+[A-Za-z0-9]+` could reduce to `[A-Za-z][A-Za-z0-9]+` although I think underscores and dashes are also allowed in usernames.  It appears that `modules/security/src/main/DataForm.scala` defines a username pattern `[\w-]+`